Subject: [PATCH] dark_title_bar
---
Index: internal/glfw/win32_window_windows.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/internal/glfw/win32_window_windows.go b/internal/glfw/win32_window_windows.go
--- a/internal/glfw/win32_window_windows.go	(revision a96153805e960ae475392305c98bc839653fb7d3)
+++ b/internal/glfw/win32_window_windows.go	(date 1750649668924)
@@ -2427,5 +2427,10 @@
 	if !_glfw.initialized {
 		return 0, NotInitialized
 	}
-	return w.platform.handle, nil
+	return w.platform.handle, setCaptionColor(w.platform.handle, 0x292929)
+}
+
+func setCaptionColor(handle windows.HWND, color uint32) error {
+	const DWMWA_CAPTION_COLOR = 35
+	return windows.DwmSetWindowAttribute(handle, DWMWA_CAPTION_COLOR, unsafe.Pointer(&color), uint32(unsafe.Sizeof(color)))
 }
